content/references/rippled-api/transaction-formats/transaction-common-fields.md

```diff
@@ -30,9 +30,11 @@ Every transaction has the same set of common fields, plus additional fields base
 
```

Comment:
Comments on this file should go to #553, not here. (This file will leave the diff after I merge that PR and rebase.)